packages/lesmis/src/components/DashboardExportModal/components/Logo.js

```diff
@@ -0,0 +1,32 @@
+import { FlexSpaceBetween } from '@tupaia/ui-components';
+import React from 'react';
+import styled from 'styled-components';
+
+const Img = styled.img`
+  height: 54px;
+`;
+
+const Container = styled(FlexSpaceBetween)`
+  top: 30px;
+  left: 50px;
+  width: 236px;
+  position: absolute;
+`;
+
+const Text = styled.p`
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 900;
+  line-height: 140%;
+  width: 190px;
+  padding-top: 6px;
+`;
+
+export const Logo = () => {
+  return (
+    <Container>
+      <Img alt="logo" src="/lesmis-logo.png" />
+      <Text>Ministry of Education and Sports Lao PDR</Text>
+    </Container>
+  );
+};
```

Comment:
This component is introduced because PDF export cannot render svg properly.

Reply:
🤔 strange, if chrome can render it normally it should be able to print it...

Reply:
Yeah, some svg functions isn't really compatible with PDF export..

Reply:
Discussed.